<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IUCalMain</class>
 <widget class="QMainWindow" name="IUCalMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>572</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>745</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>745</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IUCalMain</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="title">
     <string>Variables </string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Simulate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadDefaults">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Defaults</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>SecPerHr            </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inSecPerHr">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>222</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Base BandW idth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inBandwidth">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SecPerDay         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inSecPerDay">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="AC_OutputDays">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="AC_OutputWeeks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SampleDayOutput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="SampleWeekOutput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>20</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="title">
     <string>Samples</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inSP1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inSP2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inSP3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>107</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Simulate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BWidth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BWidth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BWidth:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Sp1BWout">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Sp2BWout">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Sp3BWout">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Sp1Toggle">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="Sp2Toggle">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="Sp3Toggle">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
